src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -162,7 +162,29 @@ private function handleRaw(Request $request, int $type = self::MASTER_REQUEST)
                 if (null === $response) {
                     $msg .= ' Did you forget to add a return statement somewhere in your controller?';
                 }
-                throw new \LogicException($msg);
+
+                $e = new \LogicException($msg);
+
+                if (!$controllerDefinition = $this->parseControllerDefinition($controller)) {
+                    throw $e;
+                }
+
+                $r = new \ReflectionProperty(\Exception::class, 'trace');
```

Comment:
What about using a rebound closure instead of reflection? It'd me things easier to read IMHO.

Reply:
> Warning: Cannot bind closure to scope of internal class Exception